### PR TITLE
Auto export addon folder

### PR DESCRIPTION
**Pull request by samselikoff**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -205,3 +219,40 @@ class FindDummyAppFiles extends Plugin {
...
+
+      addonFiles.forEach(addonFile => {
+        let module = addonFile.replace('.js', '');
+        let file = path.join(this.outputPath, moduleType, `${module}.js`);
+        ensureDirectoryExistence(file);
+        fs.writeFileSync(file, `export { default } from 'ember-cli-addon-docs/${moduleType}/${module}';`);
+      });
+    });
+
+  }
```
Comment (dfreeman) [inline index.js]: Will this handle `routes`? I saw you removed that from the `app` tree, but I'm not sure I see how that reexport gets generated here.

**Comment by dfreeman** [comment]
Looks like the test failures are caused by an update to the tokenization format used by Marked in the latest release—I just pushed a fix to master that should take care of those.

File: index.js (hunk 1 above)
Comment (samselikoff) [reply index.js]: Woops! Good catch... some acceptance tests woulda caught that

**Comment by samselikoff** [comment]
Now Ember 2.12 is failing - probably due to this line https://github.com/ember-learn/ember-cli-addon-docs/blob/master/addon/components/docs-viewer/x-main/component.js#L31

**Pull request merged**